storage/doc.go

```diff
@@ -274,15 +274,43 @@ To generate the signature, you must have:
 
 # Errors
 
-Errors returned by this client are often of the type [googleapi.Error].
-These errors can be introspected for more information by using [errors.As]
-with the richer [googleapi.Error] type. For example:
+Errors returned by this client are often of the type [googleapi.Error] or
+[github.com/googleapis/gax-go/v2/apierror.APIError].
```

Comment:
Since we mention `googleapi.Error` below in reference to HTTP, we can leave that out here. 
```suggestion
Errors returned by this client are often of the type [github.com/googleapis/gax-go/v2/apierror.APIError].
```

Note that this was accurate to out client before we supported gRPC so now we should mention both as errors unwrapped from the "higher" level apierror.APIError.